### ISSUE TITLE
Subtitles should not scroll up when there is a new line of subtitles

### ISSUE THREAD

**Issue by Grauen** [yuliskov/SmartTubeNext#668]
If subtitles are enabled I noticed this very confusing behavior :

There is a certain amount of space available for subtitle text. In my case it's like 50 percent of the screen width and 2 lines of text in height. I'll call this a "Subtitle Box". 

Now usually if there is no more space for text in a Subtitle Box, the box is cleared so no more text is visible and the next text will be shown. This is usually synced to what you can hear.

So what happens now is - the box won't be cleared completely but the 2nd row of text will move to the first row and the new text will be added to the 2nd row.

This is very confusing. You have no visual hint (like a darker background or a different color for the updated first line) and it is almost impossible to read this.

Can we introduce a setting that allows you to clear the complete subtitle box and disable scrolling?

**Comment by Grauen** [comment]
@yuliskov hi there, any chance this will be fixed?

**Comment by Grauen** [comment]
@yuliskov ping 😅

**Comment by matteo-maff** [comment]
Same issue here, with the latest version on Chromecast with Google TV. 
Auto-generated subtitles "scroll" with a one-line delay. 
This, together with issue [902](https://github.com/yuliskov/SmartTubeNext/issues/902 ) makes subtitles quite difficult to follow. 
Disabling scrolling altogether (like in the default yt app) would make it so much more readable.

**Comment by Henkow** [comment]
I also find this very confusing/hard to follow when watching. A setting to completely clear the subtitle box would be much appreciated!

**Comment by Grauen** [comment]
This was resolved quietly.
